deploy/examples/create-external-cluster-resources.py

```diff
@@ -1097,7 +1097,7 @@ def create_rgw_admin_ops_user(self):
             "--display-name",
             "Rook RGW Admin Ops user",
             "--caps",
-            "info=read;buckets=*;users=*;usage=read;metadata=read;zone=read",
+            "buckets=*;users=*;usage=read;metadata=read;zone=read",
```

Comment:
>  Now it will only add this cap if ceph version support it

How is the `info=read` optional only if ceph supports it? It appears to always be removed.

Reply:
I agree with @travisn. Even I am not seeing true of the intent of the PR. From the commit at least info=read is based on the ceph version, but here it is just removed.

Reply:
Here we are adding the check 
https://github.com/rook/rook/blob/f167271d311373ff3bca65493d1bea29092b4fd9/deploy/examples/create-external-cluster-resources.py#L1131-L1157

Reply:
here it is adding `info=read` where is the ceph version comparison, IIUC, what we are expecting is

```python3
cmd = [
            "radosgw-admin",
            "user",
            "create",
            "--uid",
            self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
            "--display-name",
            "Rook RGW Admin Ops user",
            "--caps",
        ]
if ceph_version.atleast(pacific):
          cmd.append("info=read;buckets=*;users=*;usage=read;metadata=read;zone=read")
else:
          cmd.append("buckets=*;users=*;usage=read;metadata=read;zone=read")

Reply:
@parth-gr I see, the caps attempt to be applied on lines 1137-1157 and if it's not supported we catch the exception.